addons/website/static/tests/dashboard_tests.js

```diff
@@ -0,0 +1,67 @@
+odoo.define('website/static/tests/dashboard_tests', function (require) {
+"use strict";
+
+const ControlPanel = require('web.ControlPanel');
+const Dashboard = require('website.backend.dashboard');
+const testUtils = require("web.test_utils");
+
+const { createParent, nextTick, prepareTarget } = testUtils;
+
+QUnit.module('Website Backend Dashboard', {
+}, function () {
+    QUnit.test("mounted is called once for the dashboarrd's ControlPanel", async function (assert) {
```

Comment:
double r

Reply:
I'll fix it in master before invoking fwbot